tests/test_repo_name.py

```diff
@@ -0,0 +1,117 @@
+from verinfast.utils.utils import get_repo_name_url_and_branch
+
+url_list = [
```

Comment:
prefer to move this to a separate file in fixtures.

Reply:
fixed